support/remote_health_monitoring/using-insights-operator.adoc

```diff
@@ -0,0 +1,29 @@
+:_content-type: ASSEMBLY
+[id="using-insights-operator"]
+= Using Insights Operator
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: using-insights-operator
+
+toc::[]
+
+The Insights Operator periodically gathers anonymized configuration and component failure status and, by default, reports that data every two hours to Red Hat. This information enables Red Hat to assess configuration and deeper failure data than is reported through Telemetry. Users of {product-title} can display the report in the Insights Advisor service on link:https://console.redhat.com/openshift[Red Hat Hybrid Cloud Console].
```

Comment:
Is there another word that can be used instead of "anonymized?"

Reply:
Removed, we state elsewhere how the data is obfuscated so it really isn't required here.